data/changelog/infra/2024-10-02-updates.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Infrastructure Updates
+tags: [infrastructure]
+---
+
+# Catching up
+
+[Taride](https://tarides.com/)'s OCaml Infrastructure team has been hard at work
```

Comment:
s/Taride/Tarides (I don't know if that's a typo or that how Tarides's is supposed to be written.

Reply:
Thomas decided that any possessive of Tarides should be Tarides'